Display welcome note

Currently when app is opened for first time, the home page is empty. A note which explains all features of app would be nice.

We need to run an `INSERT` query when app is installed in `lib/core/databases/sqflite_setup.dart`

Full Error:
```
Launching lib\main.dart on CPH2381 in debug mode...
Running Gradle task 'assembleDebug'...
/C:/Users/ciphe/AppData/Local/Pub/Cache/hosted/pub.dev/flutter_math_fork-0.7.1/lib/src/widgets/selection/gesture_detector_builder_selectable.dart:38:22: Error: Type 'TapDragUpDetails' not found.
  void onSingleTapUp(TapDragUpDetails details) {
                     ^^^^^^^^^^^^^^^^
/C:/Users/ciphe/AppData/Local/Pub/Cache/hosted/pub.dev/flutter_math_fork-0.7.1/lib/src/widgets/selection/gesture_detector_builder_selectable.dart:38:22: Error: 'TapDragUpDetails' isn't a type.
  void onSingleTapUp(TapDragUpDetails details) {
                     ^^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* Where:
Script 'C:\dev\tooling\flutter\packages\flutter_tools\gradle\src\main\groovy\flutter.groovy' line: 1297

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\dev\tooling\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6m 23s
Exception: Gradle task assembleDebug failed with exit code 1

```

--- a/lib/core/databases/sqflite_setup.dart
+++ b/lib/core/databases/sqflite_setup.dart
@@ -60,30 +60,23 @@ class DBProvider {
               ${NoteDependencies.ASSET_PATH} TEXT
             )
             """);
+          log.i("Inserting welcome note");
           //as soon as the tables are created, insert a welcome note
-          await db.execute("""
-            INSERT INTO ${Notes.TABLE_NAME} (
-              ${Notes.ID},
-              ${Notes.CREATED_AT},
-              ${Notes.TITLE},
-              ${Notes.BODY},
-              ${Notes.HASH},
-              ${Notes.LAST_MODIFIED},
-              ${Notes.PLAIN_TEXT},
-              ${Notes.DELETED},
-              ${Notes.AUTHOR_ID}
-            ) VALUES (
-              "",
-              "",
-              "Welcome to Dairy App",
-              "This is a welcome note. You can delete this note and start writing your own notes.",
-              "",
-              "",
-              "This is a welcome note. You can delete this note and start writing your own notes.",
-              0,
-              "guest_user_id"
-            )
-            """);
+
+          String body = """[{\"insert\":\"Welcome to DiaryVault!\\n\\nKey Features:\\n\\n-Rich text editor with support for images and videos.\\n\\n-Your data is securely preserved on your Google Drive / Dropbox account, ensuring complete ownership and privacy\\r.\\n\\n-Sync data between multiple devices.\\r\\n\\n-Fingerprint login on supported devices.\\r\\n\\n-Multiple Themes.\\n\\n\\nHappy notemaking!\\nThe DiaryVault team.\\n\\n\"}]""";
+
+          Map<String,Object> notemap = {
+            Notes.ID: "f773a170-6447-11ee-9a76-c314b6be99a3",
+            Notes.CREATED_AT: DateTime.now().millisecondsSinceEpoch,
+            Notes.TITLE: "Welcome to DiaryVault",
+            Notes.BODY: body,
+            Notes.HASH: "welcome_note_hash",
+            Notes.LAST_MODIFIED: DateTime.now().millisecondsSinceEpoch,
+            Notes.PLAIN_TEXT: "Read me to get started!",
+            Notes.DELETED: 0,
+            Notes.AUTHOR_ID: "guest_user_id",
+          };
+          await db.insert(Notes.TABLE_NAME, notemap);
 
           log.i("All create queries executed successfully");
           log.i("Welcome Note inserted into table: ${Notes.TABLE_NAME}");

--- a/lib/core/databases/sqflite_setup.dart
+++ b/lib/core/databases/sqflite_setup.dart
@@ -60,8 +60,35 @@ class DBProvider {
               ${NoteDependencies.ASSET_PATH} TEXT
             )
             """);
+          //as soon as the tables are created, insert a welcome note
+          await db.execute("""
+            INSERT INTO ${Notes.TABLE_NAME} (
+              ${Notes.ID},
+              ${Notes.CREATED_AT},
+              ${Notes.TITLE},
+              ${Notes.BODY},
+              ${Notes.HASH},
+              ${Notes.LAST_MODIFIED},
+              ${Notes.PLAIN_TEXT},
+              ${Notes.DELETED},
+              ${Notes.AUTHOR_ID}
+            ) VALUES (
+              "",
+              "",
+              "Welcome to Dairy App",
+              "This is a welcome note. You can delete this note and start writing your own notes.",
+              "",
+              "",
+              "This is a welcome note. You can delete this note and start writing your own notes.",
+              0,
+              "guest_user_id"
+            )
+            """);
 
           log.i("All create queries executed successfully");
+          log.i("Welcome Note inserted into table: ${Notes.TABLE_NAME}");
+        //   print note
+          await db.query(Notes.TABLE_NAME).then((value) => log.i(value));
         } catch (e) {
           log.e(e);
           rethrow;

